src/main/java/teammates/ui/controller/Action.java

```diff
@@ -452,12 +453,39 @@ public String getRequestParamValue(String paramName) {
     }
     
     /**
+     * Retrieves the value for the specified parameter expected to be present in the http request.
+     *
+     * @param paramName  a constant from the {@link Const.ParamsNames} class.
+     * @return the value of the specified parameter if it exists, asserts otherwise.
+     */
+    public String getExpectedRequestParamValue(String paramName) {
+        String value = HttpRequestHelper.getValueFromParamMap(requestParameters, paramName);
+        Assumption.assertPostParamNotNull(paramName, value);
+        return value;
+    }
+    
```

Comment:
How about reusing the original `getRequestParamValue` by doing:
```
public String getExpectedRequestParamValue(String paramName) {
    String value = this.getRequestParamValue(paramName);
    Assumption.assertPostParamNotNull(paramName, value);
    return value;
}
```

That way, if the code in `getRequestParamValue` needs to be changed, the changes will propagate to the new method as well.